ocean-components/src/main/java/br/com/useblu/oceands/components/compose/OceanCardOption.kt

```diff
@@ -147,7 +187,7 @@ fun OceanCardOption(
                     OceanText(
                         text = item.title ?: "",
                         style = OceanTextStyle.heading4,
-                        color = if (item.disabled) OceanColors.interfaceDarkDown else OceanColors.brandPrimaryDown
+                        color = colorTitle ?: if (item.disabled) OceanColors.interfaceDarkDown else OceanColors.brandPrimaryDown
```

Comment:
Talvez seria uma boa deixar o disabled ainda como está. Daria pra ao invés de colorTitle ser opcional e null como default, ter valor default OceanColors.brandPrimaryDown 
```kotlin
fun OceanCardOption(
...
     colorTitle: Color = OceanColors.brandPrimaryDown
...
```
e aqui:
```suggestion
                        color = if (item.disabled) OceanColors.interfaceDarkDown else colorTitle
```

Reply:
Realmente, bem melhor.

Feeeeeito